cmd/deployment-check/deployment.go

```diff
@@ -251,6 +264,30 @@ func createDeployment(ctx context.Context, deploymentConfig *v1.Deployment) chan
 	return createChan
 }
 
+// createContainerConfig creates a container resource spec and returns it.
+func createToleration(t string) corev1.Toleration {
+
+	// return blank tolerations if no value provided in yaml file
+	if len(t) == 0 {
+		tolerations := corev1.Toleration{
+			Key:      "",
+			Operator: "Exists",
```

Comment:
use the constant `corev1.NodeSelectorOpExists` here

Reply:
this changes it to an operator which breaks the code. I will use the above comment and just keep Operator